docker-compose.yml

```
@@ -0,0 +1,53 @@
+    image: redis:7
+    ports:
+      - "6379:6379"
+  minio:
+    image: minio/minio
+    command: server /data --console-address ":9001"
+    environment:
+      MINIO_ROOT_USER: admin
+      MINIO_ROOT_PASSWORD: admin12345
+    ports:
+      - "9000:9000"
+      - "9001:9001"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)